src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -209,53 +214,61 @@ public void downloadZippedFile(Environment env, HttpServletRequest request,
 
 			String type = Utils.getJSONString(contentItem, CONTENT_JSON_ATTR_TYPE);
 			String title = Utils.getJSONString(contentItem, CONTENT_JSON_ATTR_TITLE);
-
-			// files
-			if (contentUrl == null && contentUrl.length() == 0) {
+			String copyrightAlert = Utils.getJSONString(contentItem,
```

Comment:
A lot of this section is almost identical to code starting around 530.  Also that code contains the same && vs || error.  It is easier to fix problem that happen fewer times.